pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -9,12 +9,19 @@ const (
 	SecurityContextConstraintPrivileged = "privileged"
 	// SecurityContextConstraintRestricted is used as the name for the system default restricted scc.
 	SecurityContextConstraintRestricted = "restricted"
+	// SecurityContextConstraintNonRoot is used as the name for the system default non-root scc.
+	SecurityContextConstraintNonRoot = "nonroot"
+	// SecurityContextConstraintHostMount is used as the name for the system default host mount only scc.
+	SecurityContextConstraintHostMount = "hostmount"
+	// SecurityContextConstraintHostNS is used as the name for the system default host accessing scc.
+	SecurityContextConstraintHostNS = "hostaccess"
 )
 
 // GetBootstrapSecurityContextConstraints returns the slice of default SecurityContextConstraints
 // for system bootstrapping.
-func GetBootstrapSecurityContextConstraints(buildControllerUsername string) []kapi.SecurityContextConstraints {
+func GetBootstrapSecurityContextConstraints(groups map[string][]string, users map[string][]string) []kapi.SecurityContextConstraints {
```

Comment:
describe the parameters... map of group names to sccs or the other way around? make the value `sets.String`